freqtrade/persistence/trade_model.py

```diff
@@ -1351,13 +1351,20 @@ def set_custom_data(self, key: str, value: Any) -> None:
         """
         CustomDataWrapper.set_custom_data(trade_id=self.id, key=key, value=value)
 
-    def get_custom_data(self, key: str, default: Any = None) -> Any:
+    def get_custom_data(self, key: str, default: Any = None, retrieval_mode: str = "value") -> Any:
```

Comment:
I'd prefer not to overload this, but instead using a different method name.

This method is part of the strategy user interface - but i see no reason for a user to actually request this from a strategy.

Reply:
I was thinking about that but I thought that in some cases having the last updated date could be useful to check is a specific custom data is outdated.

Reply:
well that's a different topic. If you think that - it should be a separate PR explaining the change and reasoning (including updating the documentation).

For this PR - it's out of scope. Please revert.